appinventor/common/src/com/google/appinventor/common/utils/StringUtils.java

```diff
@@ -329,4 +329,25 @@ public static int countMatches(String str, String sub) {
     }
     return count;
   }
+
+  /* <p>Checks if a CharSequence is empty ("") or null.</p>
+   *
+   * <pre>
+   * StringUtils.isEmpty(null)      = true
+   * StringUtils.isEmpty("")        = true
+   * StringUtils.isEmpty(" ")       = false
+   * StringUtils.isEmpty("bob")     = false
+   * StringUtils.isEmpty("  bob  ") = false
+   * </pre>
+   *
+   * <p>NOTE: This method changed in Lang version 2.0.
+   * It no longer trims the CharSequence.
+   * That functionality is available in isBlank().</p>
+   *
+   * @param cs  the CharSequence to check, may be null
+   * @return {@code true} if the CharSequence is empty or null
+   */
+  public static boolean isEmpty(final CharSequence cs) {
```

Comment:
After having thought about it a bit, I'd like if you renamed this to `isNullOrEmpty` so it's more explicit. Imagine in some future context we have a scenario where null and empty string have different meanings; the fact that `isEmpty(null) == true` is not obvious from the name of this method (versus `isEmpty("")`).

Reply:
@ewpatton Renamed and updated the 1-line statement. FYI, if string contains all spaces, method will return false. If you like, I can trim and check